startup_scripts/280_custom_links.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.contenttypes.models import ContentType
+from extras.models import CustomLink
+from startup_script_utils import load_yaml
+import sys
+
+
+custom_links = load_yaml('/opt/netbox/initializers/custom_links.yml')
+
+if custom_links is None:
+  sys.exit()
+
+def get_content_type_id(content_type_str):
+  try:
+    return ContentType.objects.get(model=content_type_str).id
+  except ContentType.DoesNotExist:
+    print("⚠️ Error determining content type id for user declared var: {0}".format(content_type_str))
+
+for link in custom_links:
+  content_type = link.pop('content_type')
+  link['content_type_id'] = get_content_type_id(content_type)
+  if link['content_type_id'] is not None:
```

Comment:
I suggest to change the logic here as follows:

```suggestion
  if link['content_type_id'] is None:
    print("⚠️ Unable to create Custom Link '{0}': The content_type '{1}' is unknown".format(link.name, content_type))
    continue
```

This will make it obvious which custom link failed to get created and why.